terranova/commands/binds.py

```diff
@@ -305,6 +305,9 @@ def plan(
 
     # Store errors if fail_at_end
     errors = False
+    # We start with the highest possible error exit code, in the following loop we will replace it with the minimum value
+    # So if there is at least one error with exit code 1, the final exit code will be 1
+    error_exit_code = 2
```

Comment:
Do you think a link to https://developer.hashicorp.com/terraform/cli/commands/plan#detailed-exitcode could help with?

Reply:
https://github.com/elastic/terranova/pull/79/commits/69c6031db2987eb123122f9bc67a0bf15c1e9b4c